src/negotiation/maker/negotiator.ts

```diff
@@ -45,18 +57,42 @@ export class Negotiator {
   }
 
   // Below are methods related to the negotiation protocol
+
+  /**
+   * Get an {@link Order} by trading pair (e.g. ETH-BTC).
+   * @param tradingPair A trading pair (e.g. ETH-BTC).
```

Comment:
```suggestion
   * @param tradingPair A trading pair (e.g. bitcoin-bitcoin-ethereum-erc20).
```

You may want to document the format: `ledger-asset-ledger-asset`.